maxim/logger/openai/realtime.py

```
@@ -0,0 +1,845 @@
+                    stored_call = self._function_calls[call_id]
+                    function_name = stored_call.get("name", None)
+
+            # If still unknown, try parsing from arguments JSON
+            if not function_name:
+                try:
+                    args_dict = json.loads(final_arguments)
+                    # Some APIs include function name in the arguments
+                    if "name" in args_dict:
+                        function_name = args_dict["name"]
+                except Exception:
+                    pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)